### ISSUE TITLE
[Components and pattern standards] Navigation mechanisms aren't consistent across pages. (04.10.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#51874]
#### VFS team name
DGIB - My Education Benefits

#### VFS product name
VA Education Letters

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Navigation mechanisms aren't consistent across pages.

### Issue Details
When logged in as user 378 (no letters), the page H1 reads: "VA education inbox" but this does not align with URL, breadcrumbs, or the H1 of the page that does have a decision letter, or the static page in development by sitewide content team. 

### VA.gov Experience Standard
[Category Number 04, Issue Number 10](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.3_AA

### Platform Recommendation
Update H1 to read “Your VA education letter” (and see additional issue ticket recommending an update for all H1s and breadcrumbs to plural language “Your VA education letters” for future scalability).

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by sterkenburgsara** [comment]
@btruongafs @bethpottsVADEPO I want to follow-up on the page with the problematic title (H1). This feedback refers to user 378 (no letters available for download). I used the word "unautheticated" in the meeting and that is my fault - incorrect and confused everyone. To clarify, this is not a static page related to the sitewide team, it is part of the product flow. The H1 at present for users with no letters is "VA education inbox" which is not in alignment with other pages within the product. You will need to update the H1 to "Your VA education letter" to match.

**Comment by bethpottsVADEPO** [comment]
Thanks @sterkenburgsara! I had a feeling this was in the flow and the change to this one H1 just got overlooked when the name of the tool changed. I think this should be a pretty easy fix for your team to make @btruongafs. Want to let me know if there's a need to look at it together? Otherwise, thanks for fixing!

**Comment by btruongafs** [comment]
Issue resolved and deployed to staging.
